content/2-2.md

```diff
@@ -1,44 +1,46 @@
 ---
-title: "[Draft] Module 2: Navigational Menus"
-permalink: /curricula/developing-accessible-content/navigational-menus/
-ref: /curricula/developing-accessible-content/navigational-menus/
+title: "[Draft] Module 2: Menus"
+permalink: /curricula/developer-modules/menus/
+ref: /curricula/developer-modules/menus/
 lang: en
 github:
   repository: w3c/wai-curricula
   path: content/2-2.md
 license: creative-commons
+acknowledgements: /curricula/acknowledgements/
 footer: >
   <p><strong>Date:</strong> $Date</p>
-  <p><strong>Editors: </strong>Daniel Montalvo and <a href="http://www.w3.org/People/shadi/">Shadi Abou-Zahra</a>. Contributors: <a href="https://www.w3.org/WAI/EO/EOWG-members">EOWG Participants</a>. </p>
+  <p><strong>Editors:</strong> Daniel Montalvo and <a href="http://www.w3.org/People/shadi/">Shadi Abou-Zahra</a>. Contributors: <a href="https://www.w3.org/WAI/EO/EOWG-members">EOWG Participants</a>. ACKNOWLEDGEMENTS lists contributors and credits.</p>
   <p>Developed by the Education and Outreach Working Group (<a href="http://www.w3.org/WAI/EO/">EOWG</a>). Developed with support from the <a href="https://www.w3.org/WAI/about/projects/wai-guide/">WAI-Guide Project</a> funded by the European Commission (EC) under the Horizon 2020 program (Grant Agreement 822245).</p>
 navigation:
-  previous: /curricula/developing-accessible-content/structure-and-semantics/
-  next: /curricula/developing-accessible-content/images-and-graphics/
+  previous: /curricula/developer-modules/page-structure/
+  next: /curricula/developer-modules/images/
 ---
  
 ## Introduction
 {:.no-display}
 
-Courses based on this module:
+Courses based on this module should:
```

Comment:
Possible additional learning outcome along the lines of "Code menus such that menu state is communicated"

Reply:
I think this is implicitly conveyed in:

> * code fly-out menus in a way that makes them usable by people using different assistive technologies and adaptive strategies

It really deserves more clarity, so I propose:

> * code fly-out menus in a way that their state is communicated to people using different assistive technologies and adaptive strategies

Reply:
Sounds good